### PR TITLE
#295 As administrator I want to create news

### PR DESCRIPTION
**Pull request by yaroslavbilokin**

### CODE REVIEW

File: admin/src/components/news-page/News-page.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            setSnackBarMessage(res);
+            setSnackBarSeverity(SUCCESS_STATUS);
+            setSnackBarStatus(true);
+            newsLoadingStatus();
+            const newNewsItems = await newsService.getAllNews();
+            setNews(newNewsItems);
+        };
+        openSuccessSnackbar(removeNews);
+    };
+    console.log(news)
```
Comment (oleksandrhodzevych) [inline admin/src/components/news-page/News-page.js:82]: delete console.log

File: admin/src/services/Local-store-service.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// export const clearLocalStorage = () => {
+//     const nChoice = {
+//         accessToken: null,
+//         refreshToken: null,
+//         userId: null,
+//         wishlist_collection: null,
+//         products_collection: null,
```
Comment (oleksandrhodzevych) [inline admin/src/services/Local-store-service.js:7]: commented code delete

File: admin/src/components/brand-add-page/Brand-add-page.js [hunk 3]
```diff
@@ -40,6 +41,8 @@ const BrandAddPage = props => {
         setBrandName(e.target.value);
     };
 
+
+
```
[the remark this answers is not in the document]
Comment (yaroslavbilokin) [reply admin/src/components/brand-add-page/Brand-add-page.js:45]: fixed

File: admin/src/components/news-page/News-page.js (hunk 1 above)
Comment (yaroslavbilokin) [reply admin/src/components/news-page/News-page.js:82]: fixed

File: admin/src/services/Local-store-service.js (hunk 2 above)
Comment (yaroslavbilokin) [reply admin/src/services/Local-store-service.js:7]: deleted

**Pull request merged**